### DISCUSSION TITLE
When converting pdf to docx file, how to treat all flow text as body text? Acrobat may automatically decide header/footers, but I want to keep all text as body, how to realize?

### DISCUSSION THREAD

**Discussion by nissansz** [pymupdf/PyMuPDF#4166] [Q&A]
When converting pdf to docx file, how to treat all flow text as body text? Acrobat may automatically decide header/footers, but I want to keep all text as body, how to realize?

**Comment by JorjMcKie** [comment]
This is nothing to expect from PyMuPDF but - if at all - from pdf2docx.
And this package has no such feature either.

**Comment by nissansz** [comment]
Ok.
